src/server/detail/save_stages_controller.cc

```diff
@@ -265,7 +265,7 @@ void SaveStagesController::SaveRdb() {
 }
 
 void SaveStagesController::UpdateSaveInfo() {
-  double seconds = double(absl::ToInt64Milliseconds(absl::Now() - start_time_)) / 1000;
```

Comment:
maybe its better to use here ToInt64Seconds(now - start)

Reply:
I took this code from absl documentation how to do converting